catroid/src/main/java/org/catrobat/catroid/ui/fragment/ScenesListFragment.java

```diff
@@ -396,7 +396,8 @@ public void onCancel(DialogInterface dialogInterface) {
 	private void checkSceneCountAfterDeletion() {
 		ProjectManager projectManager = ProjectManager.getInstance();
 		if (projectManager.getCurrentProject().getSceneList().size() == 0) {
-			Scene emptyScene = new Scene(getActivity(), String.format(getString(R.string.default_scene_name), 1), projectManager.getCurrentProject());
+			Scene emptyScene = new Scene(getActivity(), getString(R.string.default_scene_name) + " 1", projectManager
+					.getCurrentProject());
```

Comment:
:+1: Good catch!

I wrote an email to @AyyalAwwad also addressing name enumeration yesterday. Really looking forward to his feedback. Unfortunately it's not as easy as a simple string translation with format specifiers.